test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -2214,21 +2222,18 @@ describe('the rubicon adapter', function () {
           expect(imp.video.w).to.equal(640);
           expect(imp.video.h).to.equal(480);
           expect(imp.video.pos).to.equal(1);
-          expect(imp.video.context).to.equal('instream');
           expect(imp.video.minduration).to.equal(15);
           expect(imp.video.maxduration).to.equal(30);
           expect(imp.video.startdelay).to.equal(0);
           expect(imp.video.skip).to.equal(1);
           expect(imp.video.skipafter).to.equal(15);
-          expect(imp.ext.rubicon.video.playerWidth).to.equal(640);
-          expect(imp.ext.rubicon.video.playerHeight).to.equal(480);
-          expect(imp.ext.rubicon.video.size_id).to.equal(201);
```

Comment:
we need to pass size_id in rubicon bidder params... I think I may have commented this already

Reply:
Yes and bret response was that it's the server adapter will populate it

Reply:
👍